### PR TITLE
Dynamic config for docker stack

### PR DESCRIPTION
**Pull request by EarthModule**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -17,5 +17,5 @@ RUN pip install -r requirements.txt
 EXPOSE 8008
 EXPOSE 8006
```
Comment (tompulli) [inline Dockerfile:18]: redundant?

File: Dockerfile [hunk 2]
```diff
@@ -17,5 +17,5 @@ RUN pip install -r requirements.txt
...
 
-# CMD ["python", "/code/manage.py", "runserver", "0.0.0.0:8008"]
-CMD ["python", "manage.py", "runsslserver", "--certificate", ".certs/cert.pem","--key", ".certs/key.pem", "0.0.0.0:8008"]
\ No newline at end of file
+CMD ["python", "manage.py", "runserver", "0.0.0.0:8008"]
+# CMD ["python", "manage.py", "runsslserver", "--certificate", ".certs/cert.pem","--key", ".certs/key.pem", "0.0.0.0:8008"]
```
Comment (tompulli) [inline Dockerfile:21]: no need to leave this here

File: nginx-docker.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  metax-nginx-elastic-headers-config:
+    external: True
+  metax-nginx-shared-headers-config:
+    external: True
+  metax-nginx-api-response-headers-config:
+    external: True
+  metax-nginx-static-file-headers-config:
+    external: True
+  metax-nginx-dh-param-config:
+    external: True
```
Comment (tompulli) [inline nginx-docker.yml:42]: should this still be in the same compose file now that it is working?

File: docker-compose.yml [hunk 4]
```diff
@@ -17,9 +17,7 @@ services:
       RABBIT_MQ_HOSTS: rabbitmq
       ELASTIC_SEARCH_HOSTS: elasticsearch
       METAX_DATABASE_HOST: db
-    configs:
-      - source: metax-web-config
-        target: '/code/metax_api/settings/.env'
+
```
Comment (tompulli) [inline docker-compose.yml:20]: wasn't this working fine when you discovered that the configs could be updated dynamically?

**Comment by tompulli** [comment]
Tested and working! Apache still and this is done?

File: Dockerfile (hunk 1 above)
Comment (EarthModule) [reply Dockerfile:18]: Not if developer wants to setup only the metax-web container

File: Dockerfile (hunk 2 above)
Comment (EarthModule) [reply Dockerfile:21]: Deleted

File: nginx-docker.yml (hunk 3 above)
Comment (EarthModule) [reply nginx-docker.yml:42]: I would leave it out, as metax can be run without it too

File: docker-compose.yml (hunk 4 above)
Comment (EarthModule) [reply docker-compose.yml:20]: its just extra dependency, as the stack works without it just fine

**Pull request closed without merging**